### PR TITLE
✨ New feature (new supported type premiumads for amp-auto-ads)

### PR DESCRIPTION
**Pull request by rochapablo**
https://github.com/ampproject/amphtml/issues/27666

### CODE REVIEW

**Comment by rochapablo** [comment]
@googlebot I signed it!

**Comment by lannka** [comment]
@rochapablo the CLA check is failed. did you try the "What to do if you already signed the CLA" instructions?

**Comment by rochapablo** [comment]
@googlebot I signed it!

**Comment by lannka** [comment]
@rochapablo there is also travis test failures. could you fix?

**Comment by rochapablo** [comment]
@lannka, fixed.

File: EXTENSIONS_CSS_MAP [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (lannka) [inline EXTENSIONS_CSS_MAP:1]: please revert this file

Comment (rochapablo) [reply EXTENSIONS_CSS_MAP]: https://github.com/ampproject/amphtml/pull/27667/commits/b4be3c1635e05c0aed663953fe4829a8bdd45767

**Pull request merged**